addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/handler/XiaomiSensorBaseHandler.java

```diff
@@ -71,7 +71,7 @@ void calculateBatteryLevelFromVoltage(Integer voltage) {
         voltage = Math.max(VOLTAGE_MIN_MILLIVOLTS, voltage);
         Integer battLevel = (int) ((float) (voltage - VOLTAGE_MIN_MILLIVOLTS)
                 / (float) (VOLTAGE_MAX_MILLIVOLTS - VOLTAGE_MIN_MILLIVOLTS) * 100);
-        updateState(CHANNEL_BATTERY_LEVEL, new DecimalType(battLevel));
+        updateState(CHANNEL_BATTERY_LEVEL, new QuantityType<>(battLevel, PERCENT_UNIT));
```

Comment:
Changing this does not make sense for me because the `batteryLevel` inherits from default `system.battery-level` channel type. Its default item type is `Number` which cannot handle a `QuantityType` state.

See https://www.eclipse.org/smarthome/documentation/development/bindings/thing-definition.html#channels

Reply:
Reverted this change back to DecimalType